homeassistant/components/godice/config_flow.py

```diff
@@ -0,0 +1,53 @@
+"""Config flow for godice."""
+
+from __future__ import annotations
+
+from homeassistant.components.bluetooth import BluetoothServiceInfo
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+
+from .const import DOMAIN
+
+
+class GoDiceConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Setup GoDice discovered via Bluetooth."""
+
+    VERSION = 1
```

Comment:
Can be removed

Reply:
please clarify what exactly, VERSION var ?

Reply:
Has to do with migration, see https://developers.home-assistant.io/docs/config_entries_config_flow_handler/#config-entry-migration

I was told it can be removed since `VERSION = 1` is the default